src/bundle.js

```diff
@@ -2,6 +2,7 @@ import bar from './charts/bar.js';
 import donut from './charts/donut.js';
 import legend from './charts/legend.js';
 import line from './charts/line.js';
+import loading from './charts/helpers/loading.js';
```

Comment:
Taking this issue: https://github.com/eventbrite/britecharts/issues/422 into account, maybe it would make sense to call the helper 'load'? What do you think?

Reply:
Hm, what do you think about `loadingState`? I think `load` implies that it's doing something aside from providing the loadingStates.

Reply:
cool!